sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTest.java

```diff
@@ -2379,6 +2379,37 @@ public void analyzePiiEntityRecognitionWithCategoriesFilters(HttpClient httpClie
         );
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
+    public void analyzePiiEntityRecognitionWithDomainFilters(HttpClient httpClient,
+        TextAnalyticsServiceVersion serviceVersion) {
+        client = getTextAnalyticsClient(httpClient, serviceVersion);
```

Comment:
also, is this method working for you? I opened a bug against the service than in preview.5 analyze PII domain always throws 500

Reply:
ahh I see u added the link. never mind then

Reply:
Noop. I disabled the test.